Jenkinsfile.oke

```diff
@@ -281,7 +281,7 @@ EOF
                         DOCKERHUB_PASSWORD = credentials("${dockerhub_password_creds}")
                     }
                     steps {
-                        sh 'echo ${DOCKERHUB_PASSWORD} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin'
+                        # sh 'echo ${DOCKERHUB_PASSWORD} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin'
```

Comment:
Remove code, do not comment it out.  Git provides the history needed to recover if we ever need to do so.